app/infrastructure/clients/base_http_client.py

```
@@ -0,0 +1,98 @@
+                "export_state()는 인증된 세션에서만 호출할 수 있습니다."
+            )
+        return json.dumps(serialize_cookies(self._session.cookie_jar)).encode("utf-8")
+
+    async def import_state(self, state: bytes) -> None:
+        """
+        bytes 형태로 제공된 쿠키를 복원하여 새로운 세션을 시작합니다.
+        """
+        try:
+            records = json.loads(state)
+            if not isinstance(records, list):
+                raise ValueError("cookie records는 list여야 합니다.")
```

Ruff (0.15.14)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)